src/SlothApi/Services/Internal/ISlothApi.cs

```diff
@@ -26,5 +26,5 @@ public interface ISlothApi
     Task<Response<bool>> ValidateFinancialSegmentString([Path] string id);
 
     [Post("transactions")]
-    Task<HttpResponseMessage> CreateTransaction([Body] CreateTransactionViewModel transaction);
+    Task<Response<Transaction>> CreateTransaction([Body] CreateTransactionViewModel transaction);
```

Comment:
What happens if the response is an error not the transaction?

Reply:
The attribute `[AllowAnyStatusCode]` tells RestEase to not throw an exception. `ApiResult.Success` will be `false` and `ApiResult.Message` will have information if available.